include/wx/generic/bmpcbox.h

```diff
@@ -83,7 +83,7 @@ class WXDLLIMPEXP_ADV wxBitmapComboBox : public wxOwnerDrawnComboBox,
                 const wxString& name = wxASCII_STR(wxBitmapComboBoxNameStr));
 
     virtual ~wxBitmapComboBox();
-
+    virtual wxString GetStringSelection() const wxOVERRIDE;
```

Comment:
Nitpick, but this doesn't read very clearly without some space:
```suggestion

    virtual wxString GetStringSelection() const wxOVERRIDE;

```